RELEASE.md

```diff
@@ -1,8 +1,10 @@
-# Upcoming Release
```

Comment:
Keep the heading here as it was, it should be changes during the release so the automatic release notes extraction works properly!